### ISSUE TITLE
Support analytics 

### ISSUE THREAD

**Issue by yassinebridi** [redwoodjs/graphql#389]
Analytics is surely something you can't ignore when building any size of applications.
Redwood could utilize other good analytics libraries under the hood, like: 

- https://github.com/cube-js/cube.js
- https://getanalytics.io

or may be implementing it's own.

the problem with suggested libraries, is the deployment part. since redwood is planning to go full JAMstack.

**Comment by cannikin** [comment]
We like Netlify Analytics because it a) doesn't creepily track users across sites and b) should be more accurate than browser/javascript-based tracking since it's implemented on the server-side and isn't subject to ad/script blocking.

**Comment by yassinebridi** [comment]
Netlify Analytics looks really good, but isn't it limiting to use a paid solution.
I know 9$/month doesn't seem that much, but it can ramp up quickly if you have multiple sites.

_I never understood netlify's pricing, it's as free as it can get, but when you need to upgrade it's a lot 🤣_

**Comment by lachlanjc** [comment]
Yep, can highly recommend Netlify Analytics for general Redwood site analytics. It gives useful overall stats without any privacy issues:
![screenshot of it](https://user-images.githubusercontent.com/5074763/78519106-47186b00-7790-11ea-8b6b-22d97afb4c50.png)

**Comment by jtoar** [comment]
Also see https://github.com/redwoodjs/redwood/issues/605.
